src/content/guides/lazy-loading.md

```diff
@@ -101,4 +101,5 @@ Many frameworks and libraries have their own recommendations on how this should
 
 - React: [Code Splitting and Lazy Loading](https://reacttraining.com/react-router/web/guides/code-splitting)
 - Vue: [Lazy Load in Vue using Webpack's code splitting](https://alexjoverm.github.io/2017/07/16/Lazy-load-in-Vue-using-Webpack-s-code-splitting/)
+- Angular: [Lazy Loading route configuration](https://angular.io/guide/router#lazy-loading-route-configuration)
```

Comment:
Maybe link to https://angular.io/guide/router#milestone-6-asynchronous-routing instead , to give readers some context.